pkg/controller/initializer.go

```diff
@@ -13,6 +13,7 @@ func init() {
 	addManager := func(mgr manager.Manager) error {
 		k8sService := GetInstance(mgr)
 		reconciler := kieapp.Reconciler{Service: &k8sService}
+		deployConsole(&reconciler)
```

Comment:
I'm thinking we only really want this to deploy in the event we're running the operator as a deployment in openshift. We already do something similar for creating the env ConfigMaps here:
https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kieapp/kieapp_controller.go#L47

The other big benefit of moving this method call to that function instead is we can set an ownerRef of that main operator deployment on the console's pod/service/route. Thoughts? 

I expect we could also do away with the IMAGE var but just parsing the deployment object and grabbing the image setting dynamically.

Reply:
@tchughesiv These changes are now pushed